radixdlt/src/integration/java/com/radixdlt/integration/distributed/MockedPersistenceStoreModule.java

```diff
@@ -50,5 +51,10 @@ public void close() {
 		public void save(VerifiedVertexStoreState vertexStoreState) {
 			// Nothing to do here
 		}
+
+		@Override
+		public void save(Transaction transaction, VerifiedVertexStoreState vertexStoreState) {
```

Comment:
We've used `Transaction  tx` everywhere else, so might be good to be consistent here.